spec/ulid_spec.cr

```diff
@@ -56,5 +56,36 @@ describe ULID do
         (ulid_2 < ulid_1).should be_true
       end
     end
+
+    it "validator should return true if a string is a valid ulid" do
+      validator = ULID.valid?("01B3EAF48P97R8MP9WS6MHDTZ3")
+      validator.should eq true
+    end
+
+    it "should not valdate invalid strings" do
```

Comment:
Pop and empty string check in here too - good base / edge case to provide test coverage for.